src/component/mxgraph/style/style-updater.ts

```diff
@@ -84,7 +84,7 @@ export class StyleUpdater {
 
   resetStyle(bpmnElementIds: string | string[]): void {
     this.graph.batchUpdate(() => {
-      if (bpmnElementIds) {
+      if (bpmnElementIds || bpmnElementIds == '') {
```

Comment:
**praise:** we may introduce a function to manage this check
It could improve readability while eliminating duplication (which is not really mandatory here).